app/assets/javascripts/discourse/app/components/user-menu/notifications-list.js

```diff
@@ -1,6 +1,9 @@
 import UserMenuItemsList from "discourse/components/user-menu/items-list";
 import I18n from "I18n";
 import { action } from "@ember/object";
+import { ajax } from "discourse/lib/ajax";
+import { postRNWebviewMessage } from "discourse/lib/utilities";
```

Comment:
We have some abbreviation being used here and I'm not quite sure what it means.

Reply:
It looks like RN stands for React Native:

https://github.com/discourse/discourse/blob/53dd9b0c66d5f674fb69b802f06820189ed7dfbc/app/assets/javascripts/discourse/app/lib/utilities.js#L456-L460

Reply:
Oooo this is for the Discourse Mobile App... gotcha